### PR TITLE
[EDU-4440] feat: add Azion Starter Kit guide

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Related issue:

[EDU-4440]

### Changes

- Add Azion Starter Kit guide - EN/PT
- Add link in Guides page and Reference doc.
- 
### Additional links

n/a.

[EDU-4440]: https://aziontech.atlassian.net/browse/EDU-4440?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/en/pages/guides/marketplace/templates/azion-starter-kit.mdx [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+{
+  "threshold": 10,
+  "action": "allow",
+  "internal_logs": 2
+}
+```
+
+Where:
+
+- `threshold`: the maximum score the request can reach before the function takes an action. If it has no value, the function won't take action. Default value: 10.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/marketplace/templates/azion-starter-kit.mdx:102]:
```suggestion
- `threshold`: the maximum score the request can reach before the function takes an action. If it has no value, the function won't take action. Default value: `10`.
```

File: src/content/docs/en/pages/guides/marketplace/templates/azion-starter-kit.mdx [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+- `action`: the action to be taken by the function whenever the request's score is greater than or equal to the defined threshold. Possible values: allow, custom_html, deny, drop, hold_connection, random_delay, and redirect. Default value: allow.
+- `internal_logs`: every execution of the Bot Manager writes report logs, no matter the score of the request.
+
+The logs can be checked in [Real-Time Events](/en/documentation/products/observe/real-time-events/). By analyzing the logs generated by Bot Manager, you can get insights to understand if any changes in the function Instance's JSON Args are needed. You can also use Azion [Data Stream](/en/documentation/products/observe/data-stream/) as an observability tool. Read more on [How to observe logs and calibrate rules](/en/documentation/products/guides/secure/manage-bots/#use-real-time-events-to-observe-and-calibrate-rules).
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/marketplace/templates/azion-starter-kit.mdx:106]:
```suggestion
The logs can be checked in [Real-Time Events](/en/documentation/products/observe/real-time-events/). By analyzing the logs generated by Bot Manager, you can get insights to understand if any changes in the function instance's JSON Args are needed. You can also use Azion [Data Stream](/en/documentation/products/observe/data-stream/) as an observability tool. Read more on [How to observe logs and calibrate rules](/en/documentation/products/guides/secure/manage-bots/#use-real-time-events-to-observe-and-calibrate-rules).
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/azion-starter-kit.mdx [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+
+Agora você pode [gerenciar e ajustar as configurações](#gerencie-o-template) através do Azion Console.
+
+### Regras e configurações
+
+A edge application implantada inclui as seguintes regras e configurações:
+
+- **Criteria**: if `${uri}` *starts with* `/`.
+- **Behavior**: then [Set Origin](/pt-br/documentacao/produtos/build/edge-application/rules-engine/#set-origin) as `Default Origin`.
+- Para otimização e cache de imagens, as extensões de arquivo que podem ser aprimoradas com essas configurações são `.jpg`, `.jpeg`, `.bmp`, `.ico`, `.gif`, `.png`.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/azion-starter-kit.mdx:82]: ficou confusa essa frase

File: src/content/docs/pt-br/pages/guias/marketplace/templates/azion-starter-kit.mdx [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+- Para conteúdo estático, o template suporta as extensões de arquivo estático comuns. 
+- Além disso, o template:
+    - Respeita as configurações de cache da origem (ou usa um TTL padrão para cache).
+    - Fornece regras que alteram as configurações de cache para arquivos estáticos com base na extensão do arquivo solicitado para melhorar a entrega.
+
+Additionally, the edge firewall enhances security through the [Azion IP Tor Exit Nodes](/en/documentation/products/guides/secure/block-tor-networks/) network list and Azion [Bot Manager](/en/documentation/products/guides/bot-manager/).
+
+The function instance for Bot Manager has the following JSON Args:
+
+
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/azion-starter-kit.mdx:92]:
```suggestion
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/azion-starter-kit.mdx [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+{
+  "threshold": 10,
+  "action": "allow",
+  "internal_logs": 2
+}
+```
+
+Onde:
+
+- `threshold`: a pontuação máxima que a requisição pode atingir antes que a função execute uma ação. Se não tiver valor, a função não executará ação nenhuma. *Valor padrão: 10*.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/azion-starter-kit.mdx:107]:
```suggestion
- `threshold`: a pontuação máxima que a requisição pode atingir antes que a função execute uma ação. Se não tiver valor, a função não executará ação nenhuma. Valor padrão: `10`.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/azion-starter-kit.mdx [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+- `action`: a ação a ser tomada pela função sempre que a pontuação da requisição for maior ou igual ao limite definido. Valores possíveis: allow, custom_html, deny, drop, hold_connection, random_delay e redirect. *Valor padrão: allow*.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/azion-starter-kit.mdx:108]:
```suggestion
- `action`: a ação a ser tomada pela função sempre que a pontuação da requisição for maior ou igual ao limite definido. Valores possíveis: `allow`, `custom_html`, `deny`, `drop`, `hold_connection`, `random_delay` e `redirect`. Valor padrão: `allow`.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/azion-starter-kit.mdx [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+
+Agora, seu projeto está pronto para trabalhar com um fluxo de trabalho de implantação contínua, atualizando instantaneamente quaisquer alterações no aplicativo ou no repositório.
+
+### Adicione um domínio personalizado
+
+A edge application criada tem um domínio Azion atribuído para torná-la acessível através do navegador. O domínio tem o seguinte formato: `xxxxxxxxxx.map.azionedge.net`. No entanto, você pode adicionar um domínio personalizado para que os usuários acessem sua aplicação por meio dele.
+
+import Button from '~/components/Button.astro';
+
+<Button href="/pt-br/documentacao/produtos/build/edge-application/domains/" text="saiba mais sobre domínios" variant="secondary" />
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/azion-starter-kit.mdx:170]: mudar pra link do guia?

**Review by hannah-sk — APPROVED** [review]

File: src/content/docs/pt-br/pages/guias/marketplace/templates/azion-starter-kit.mdx [hunk 8]
```diff
@@ -0,0 +1,173 @@
...
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/azion-starter-kit.mdx:165]:
```suggestion
<Button href="/pt-br/documentacao/produtos/guias/configurar-dominio/" text="saiba mais sobre domínios" variant="secondary" /> 
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/azion-starter-kit.mdx [hunk 9]
```diff
@@ -0,0 +1,173 @@
...
+
+Agora você pode [gerenciar e ajustar as configurações](#gerencie-o-template) através do Azion Console.
+
+### Regras e configurações
+
+A edge application implantada inclui as seguintes regras e configurações:
+
+- **Criteria**: if `${uri}` *starts with* `/`.
+- **Behavior**: then [Set Origin](/pt-br/documentacao/produtos/build/edge-application/rules-engine/#set-origin) as `Default Origin`.
+- Para otimização e cache de imagens, as extensões de arquivo que podem ser aprimoradas são `.jpg`, `.jpeg`, `.bmp`, `.ico`, `.gif`, `.png`.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/azion-starter-kit.mdx:82]: ajustei aqui. @hannah-sk , acha que fica melhor?

**Pull request merged**